scripts/qa_crowbarsetup.sh

```diff
@@ -4443,6 +4454,42 @@ function onadmin_run_cct()
     return $ret
 }
 
+function onadmin_devsetup()
+{
+    # install dev setup dependencies
+    add_sdk_repo
+    ensure_packages_installed git gcc ruby2.1-devel sqlite3-devel libxml2-devel
+
+    # create development folders
+    mkdir -p /opt/crowbar/crowbar_framework/db /opt/crowbar/barclamps
+
+    # copy existing database
+    cp /opt/dell/crowbar_framework/db/production.sqlite3 /opt/crowbar/crowbar_framework/db/development.sqlite3
+
+    # clone git repos
```

Comment:
We need this git clones to use guard to setup the initial folder structure which is needed to install all GEM's and the barclamps.